cmd/describe/cluster/cmd.go

```diff
@@ -134,22 +134,26 @@ func run(_ *cobra.Command, argv []string) {
 		reporter.Errorf("Failed to parse creator ARN for cluster '%s'", clusterKey)
 		os.Exit(1)
 	}
-	phase := ""
 
-	if cluster.State() == cmv1.ClusterStatePending {
-		phase = "(Preparing account)"
+	// Find the details of the shard
+	shardPath, err := clustersCollection.Cluster(cluster.ID()).ProvisionShard().Get().Send()
+	var shard string
+	if shardPath != nil && err == nil {
+		shard = shardPath.Body().HiveConfig().Server()
 	}
 
-	if cluster.State() == cmv1.ClusterStateInstalling {
-		if !cluster.Status().DNSReady() {
-			phase = "(DNS setup in progress)"
-		}
+	phase := ""
+	if cluster.State() == cmv1.ClusterStatePending {
+		phase = "(Preparing account)"
+	} else if cluster.State() == cmv1.ClusterStateInstalling {
 		if cluster.Status().ProvisionErrorMessage() != "" {
 			errorCode := ""
 			if cluster.Status().ProvisionErrorCode() != "" {
 				errorCode = cluster.Status().ProvisionErrorCode() + " - "
 			}
 			phase = "(" + errorCode + "Install is taking longer than expected)"
+		} else if !cluster.Status().DNSReady() {
+			phase = "(DNS setup in progress)"
```

Comment:
Did you want to including changing the install phase in this PR?

Reply:
It's a code reorg, but yeah I should remove it, or at least have it as a separate commit. I can do that right away

Reply:
removed